build_all/configs/ulib_functions.cmake

```diff
@@ -110,6 +110,11 @@ function(ulib_populate_test_target target_name)
             USE_CTEST
     )
 
+    set_target_properties(${target_name}
+        PROPERTIES
+            FOLDER "uLib Tests"
```

Comment:
>uLib Tests [](start = 20, length = 10)

Is there any problem using space and uppercase characters? isn't it better to keep snake case?

Reply:
I think it looks nice with the regular uppercase and spaces. It's a directory only generated which Visual Studio knows about. It has nothing to do with the build directories such as `cmake/azure_ulib_c`. It an organizational property only for VS

Reply:
Sounds good.

---
In reply to: [334695109](https://github.com/Azure/azure-ulib-c/pull/38#discussion_r334695109) [](ancestors = 334695109)